protocol/server_parameters.go

```diff
@@ -2,6 +2,10 @@ package protocol
 
 import "time"
 
+// NonForwardSecurePacketSizeReduction is the number of bytes a non forward-secure packet has to be smaller than a forward-secure packet
+// This makes sure that those packets can always be retransmitted without splitting the contained StreamFrames
+const NonForwardSecurePacketSizeReduction = 50
```

Comment:
I would prefer to not have a magic number, but construct this value out of other constants we already have.